csharp/ql/src/change-notes/2023-04-20-xss-query-tainttracking.md

```diff
@@ -0,0 +1,4 @@
+---
+category: majorAnalysis
+---
+* C#: Extending `TaintTrackingConfiguration` in `XSSQuery.qll` no longer affects query results.
```

Comment:
I don't think this change note is needed, TBH. It was never intended for 3rd parties to customize the query by overriding the configuration itself; instead, they could add new sources or sinks.

Reply:
Alright - I will go ahead and remove it.